assets/shell-quote.js

```diff
@@ -210,7 +210,6 @@ function parse(s, env, opts) {
       }
       else if (/[*@#?$!_\-]/.test(s.charAt(i))) {
         varname = s.charAt(i);
-        i += 1;
```

Comment:
@aman-v-singh Can we mention more as to why we're removing this? In RCA it's only mentioned this was added unnecessarily. Although, the `shell-quote` library still has this same line present. Also, it seems character after `$` was not getting removed and it was 1 char after $ that's getting removed.

You can try to better understand the corresponding function and what it does and reason out the removal. I'm afraid removing this might break other stuff.

https://github.com/ljharb/shell-quote/blob/main/parse.js#L134

Reply:
@VShingala I realised that the real issue is caused because the `$` char needs to be escaped,
So doing something like:- `curl -X GET "https://alextest.com/" -H "5f005074: [jKanrLih\$!]SooCV5u)aI!pt9"`
or using single quotes:- `curl -X GET "https://alextest.com/" -H '5f005074: [jKanrLih$!]SooCV5u)aI!pt9'`
would solve the problem.

So no code change is required, I will reply to the reporter about this fix for the issue.

If you agree this can be closed.